cpp/include/Ice/Communicator.h

```diff
@@ -121,23 +111,22 @@ namespace Ice
             }
         }
 
-        /// Convert a proxy to a set of proxy properties.
+        /// Converts a proxy into a set of proxy properties.
         /// @param proxy The proxy.
         /// @param property The base property name.
         /// @return The property set.
         PropertyDict proxyToProperty(const std::optional<ObjectPrx>& proxy, std::string property) const;
 
-        /// Convert an identity into a string.
+        /// Converts an identity into a string.
         /// @param ident The identity to convert into a string.
         /// @return The "stringified" identity.
         [[nodiscard]] std::string identityToString(const Identity& ident) const;
 
-        /// Create a new object adapter. The endpoints for the object adapter are taken from the property
-        /// <code><em>name</em>.Endpoints</code>.
+        /// Creates a new object adapter. The endpoints for the object adapter are taken from the property
+        /// `name.Endpoints`.
         /// It is legal to create an object adapter with the empty string as its name. Such an object adapter is
         /// accessible via bidirectional connections or by collocated invocations that originate from the same
```

Comment:
can we remove the `that originate from the same....` this is the definition of collocared.